transfer.md

```diff
@@ -58,36 +60,38 @@ There are other equivalents for these tools
 | `docker rmi`    | [`podman rmi`](./docs/podman-rmi.1.md)        |
 | `docker run`    | [`podman run`](./docs/podman-run.1.md)        |
 | `docker save`   | [`podman save`](./docs/podman-save.1.md)      |
+| `docker start`  | [`podman start`](./docs/podman-start.1.md)    |
 | `docker stop`   | [`podman stop`](./docs/podman-stop.1.md)      |
 | `docker tag`    | [`podman tag`](./docs/podman-tag.1.md)        |
+| `docker top`    | [`podman top`](./docs/podman-top.1.md)        |
 | `docker unpause`| [`podman unpause`](./docs/podman-unpause.1.md)|
 | `docker version`| [`podman version`](./docs/podman-version.1.md)|
 | `docker wait`   | [`podman wait`](./docs/podman-wait.1.md)   |
 
-*** Use `podman exec` to enter a container and `podman logs` to view the output of pid 1 of a container.
 **** Use mount to take advantage of the entire linux tool chain rather then just cp.  Read [`here`](./docs/podman-cp.1.md) for more information.
 
 ## Missing commands in podman
 
 Those Docker commands currently do not have equivalents in `podman`:
 
- * `docker container`
- * `docker events`
- * `docker image`
- * `docker network`
- * `docker node`
- * `docker plugin`
- * `docker port`
- * `docker rename`
- * `docker restart`
- * `docker search`
- * `docker secret`
- * `docker service`
- * `docker stack`
- * `docker swarm`
- * `docker system`
- * `docker top`
- * `docker volume`
+| Missing command | Description|
+| :--- | :--- |
+| `docker container`||
+| `docker events`   ||
+| `docker image`    ||
+| `docker network`  ||
+| `docker node`     ||
+| `docker plugin`   |podman does not support plugins.  We recommend you use alternative OCI Runtimes or OCI Runtime Hooks to alter behaviour of podman.|
+| `docker port`     ||
+| `docker rename`   | podman does not support rename, you need to podman rm/podman create to rename a container.|
```

Comment:
suggest "need to podman" to "need to use podman"

Reply:
Fixed